specification/carbon/Carbon.Management/client.tsp

```diff
@@ -50,5 +66,75 @@ using Microsoft.Carbon;
   "ResourceGroupCarbonEmissionTopItemMonthlySummary",
   "csharp"
 );
-@@clientName(Microsoft.Carbon, "CarbonOptimizationMgmt", "python");
-@@clientName(Microsoft.Carbon, "CarbonOptimizationManagementClient", "java");
+@@clientName(CarbonEmissionDataAvailableDateRange,
+  "CarbonEmissionAvailableDateRange",
+  "csharp"
+);
+@@clientName(CategoryTypeEnum, "CarbonEmissionCategoryType", "csharp");
+@@clientName(EmissionScopeEnum, "CarbonEmissionScope", "csharp");
+@@clientName(AccessDecisionEnum, "CarbonEmissionAccessDecision", "csharp");
+@@clientName(ReportTypeEnum, "CarbonEmissionQueryReportType", "csharp");
+@@clientName(DateRange, "CarbonEmissionQueryDateRange", "csharp");
+@@clientName(DateRange.start, "StartOn", "csharp");
```

Comment:
I think we already have this renaming logic in the generator [InputRenameTimeToOnTransformer](https://github.com/Azure/autorest.csharp/blob/main/src/AutoRest.CSharp/Common/Input/InputTypes/Transformation/InputRenameTimeToOnTransformer.cs#L12)
what did you get without this? is this start type DateTime?

Reply:
unfortunately, not work without this rename.